tests/unit/schema/test_api.py

```diff
@@ -344,6 +346,23 @@ def test_messages_nested():
 def test_services():
     L = descriptor_pb2.SourceCodeInfo.Location
 
+    # Set up retry information.
+    def _n(method_name: str):
+        return {
+            'service': 'google.example.v2.FooService',
+            'method': method_name,
+        }
```

Comment:
Nit: extra newline please

Reply:
Done, and added comments. (I have a hard rule that a comment must follow any blank line within a function.)

Reply:
Nice rule! Think I'll steal it.